Corrects spelling mistakes

--- a/client_announce.c
+++ b/client_announce.c
@@ -865,8 +865,8 @@
     if (group_list[listidx].restart) {
         if (group_list[listidx].sync_mode) {
             log0(group_list[listidx].group_id, 0,
-                    "Sync mode and restart mode incompatable");
-            send_abort(listidx, "Sync mode and restart mode incompatable");
+                    "Sync mode and restart mode incompatible");
+            send_abort(listidx, "Sync mode and restart mode incompatible");
             return;
         }
         read_restart_file(listidx);
